docs/source/reference/cli.rst

```diff
@@ -79,13 +79,43 @@ by passing ``--skip-config`` flag to the CLI as shown below:
 Authentication and auth token caching
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you specify username and password as authentication credentials in the
-configuration file, the CLI will try to use those credentials to authenticate and
-retrieve an auth token.
+If you don't wish to store your password in plain-text as shown in the previous section,
```

Comment:
Also, I think we should probably still keep part of this paragraph somewhere here explaining how authentication works in the CLI if username and password are specific in the config.